what4/src/What4/Interface.hs

```diff
@@ -180,6 +180,10 @@ module What4.Interface
     -- * Exceptions
   , InvalidRange(..)
 
+    -- * Bitvector utilities
+  , bvZero
+  , bvOne
```

Comment:
Perhaps it makes more sense to include this under the "Utility combinators" Haddock section above as a sub-section.

Reply:
Good call